build-logic/binary-compatibility/src/main/groovy/gradlebuild/binarycompatibility/transforms/FindGradleJars.groovy

```diff
@@ -30,13 +30,16 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.InputFiles
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
+import org.gradle.work.DisableCachingByDefault
 
 @CompileStatic
+@DisableCachingByDefault(because = "Only filters the input artifact")
 abstract class FindGradleJars implements TransformAction<Parameters> {
 
     @CompileStatic
     interface Parameters extends TransformParameters {
         @InputFiles
+        @PathSensitive(PathSensitivity.NAME_ONLY)
```

Comment:
Do we need to mention the change around how artifact transform warnings are handled now in the release notes?

Reply:
Good question. I can add something to the upgrading doc.

Reply:
I tried adding something to the upgrading doc. While writing it I came to the conclusion that it is probably better not to include it. I'd treat this as a new validation which we added with `enableStricterValidation`. We don't consider that a possibly breaking change, right?